sqlglot/parser.py

```diff
@@ -2120,22 +2120,28 @@ def _parse_property(self) -> t.Optional[exp.Expression]:
 
         return self.expression(exp.Property, this=key, value=value)
 
-    def _parse_stored(self) -> exp.FileFormatProperty:
-        self._match(TokenType.ALIAS)
```

Comment:
I would consider refactoring this so this goes first:

```python
        if self._match_texts("BY"):
            return self.expression(exp.StorageHandlerProperty, this=self._parse_var_or_string())
```

You can then avoid the big diff, thus lowering the reviewing mental overhead.